apix_documentation_swagger_v3.yaml

```diff
@@ -168,6 +168,17 @@ info:
 
           `-H 'Accept: application/vnd.regalii.v3.2+json'`
 
+    # Pagination
+
+    For those endpoints that return a list of resources, we have added the
+    `X-Pagination` header on the response to make pagination easier. It looks
+    something like this:
+
+    ```
+    X-Pagination: {"total_entries":411,"total_pages":9,"previous_page":null,"current_page":1,"next_page":2}
+    ```
+
+    __NOTE:__ This custom header does not appear in the Swagger's response
```

Comment:
Instead of `in the Swagger's response` It isn't `in Swagger response` ?

Reply:
What about: "the response obtained through Swagger"?

Reply:
Or maybe "the response obtained through this documentation"? Because I don't think the customers necessarily know that it's generated using Swagger.

Reply:
👍 for the last one